src/Windows/R/Components/Impl/Plots/Implementation/ViewModel/RPlotHistoryViewModel.cs

```diff
@@ -105,9 +105,7 @@ public IEnumerable<IRPlotHistoryEntryViewModel> SelectedPlots
                 // This can happen if an existing plot was resized too small.
                 // Don't overwrite the history image in this case, because the
                 // existing image in history will be better than no image at all.
-                if (plotImage != null) {
-                    entry.PlotImage = plotImage;
-                }
+                entry.PlotImage = plotImage;
```

Comment:
Do you want to override non-null image in history to null?

Reply:
It is now checked by the caller

Reply:
Ok